services/cache/lru.go

```diff
@@ -186,47 +185,57 @@ func (l *LRU) set(key string, value interface{}, ttl time.Duration) error {
 }
 
 func (l *LRU) get(key string, value interface{}) error {
-	if ent, ok := l.items[key]; ok {
-		e := ent.Value.(*entry)
+	e, err := l.criticalGet(key)
```

Comment:
I would suggest renaming it to `getItem`. That's what it essentially does.

Reply:
I have updated PR with name change.
Since its an internal function only used here, I can move this to anonymous function too.
Let me know what works best.

Reply:
This is fine. Thanks!